dev/core/docker-compose.yml

```diff
@@ -56,6 +56,8 @@ services:
       - ./config/:/etc/assemblyline/
       - ../../../:/opt/alv4/
     command: python3 /opt/alv4/assemblyline-core/assemblyline_core/alerter/run_alerter.py
+    healthcheck:
```

Comment:
I'm not too familiar with docker-compose and how it works, but this healthcheck block seems very repetitive and looks like it is applied to nearly every container in this file. Is there a way such that it is applied as default to containers and requires explicit declaration if it is not to be applied to a container?

Reply:
The other place it can come from is the container definition, so the other place it can be added is the assemblyline_dev dockerfile. Then it can be overwritten for cases that don't use that check.

Reply:
Ah okay, and is this a more optimal approach?